include/subnet_cadmium.hpp

```diff
@@ -116,7 +117,7 @@ class Subnet {
         }
 
         friend std::ostringstream& operator<<(std::ostringstream& os,
-		                const typename Subnet<TIME>::state_type& i) {
+            const typename Subnet<TIME>::state_type& i) {
             os << "index: " << i.index << " & transmiting: " << i.transmiting; 
```

Comment:
My suggestion would be not to give any space before and after << operator. 
i.e., os<<"ack_num: "<<i.ack_num;

Reply:
I think it is OK to have spaces around << for readability